Changes Some Job Hud Sprites

nmajask opened the pull request:
# General Documentation

### Intent of your Pull Request

Changes Paramedic, Psychiatrist, Signal Technician, Tourist, and Clerk hud sprites to be more consistent with other hud sprites, here are the new ones:
![Paramedic Icon](https://user-images.githubusercontent.com/14363906/118419647-9e268a00-b68a-11eb-92b3-6fd75d8ac0d2.png)
![Psychiatrist](https://user-images.githubusercontent.com/14363906/118419729-daf28100-b68a-11eb-9b59-96448c9d7478.png)
![Signal Technician](https://user-images.githubusercontent.com/14363906/118421094-5c97de00-b68e-11eb-9414-a3ee8d1db1d4.png)
![tourist](https://user-images.githubusercontent.com/14363906/118423498-1db85700-b693-11eb-9892-808fa99f1a18.png)
![clerk](https://user-images.githubusercontent.com/14363906/118423501-201ab100-b693-11eb-9940-6c030deb9c33.png)



### Why is this change good for the game?

I feel the new sprites fit the game and other HUD sprites better, but it is personal preference 

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

It changes some sprites to look better

### What should players be aware of when it comes to the changes your PR is implementing?

Some hud sprites will change a bit

### What general grouping does this PR fall under? 

Sprite Changes

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

Also no

# Changelog

:cl:    
imageadd: added some new hud icons  
imagedel: deleted some of the old hud icons 
/:cl:

nmajask committed:
Add files via upload

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
changed sigtech, tourist, and clerk icons

added more detail to sigtech icon
new tourist and clerk icons

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
Some more icon changes

I uploaded the wrong file

nmajask committed:
Merge branch 'yogstation13:master' into master

nesisknesis reviewed (changes requested):
Itd probably not bother most players, but the shading on the Paramedic icon looks a little blurry in game with the way youve shaded it, and I think i've got a better design for a the Sig Tech icon that flows more like a radio signal symbol. The Clerk icon though is pretty uninspired.
![image](https://user-images.githubusercontent.com/37936685/118642510-5e1eee80-b7d3-11eb-9e7e-183af6c4d40b.png)

Here's my edits of them. I've used solely flat colour so that its more plain and bold, and the Clerk icon i've reverted back to the old.
![image](https://user-images.githubusercontent.com/37936685/118644132-5a8c6700-b7d5-11eb-9962-5de4e85fc399.png)
raw:
![hudclerk](https://user-images.githubusercontent.com/37936685/118660354-08a00d00-b7e6-11eb-94d5-82667fd1c455.png)
![hudpara](https://user-images.githubusercontent.com/37936685/118660384-0fc71b00-b7e6-11eb-9b8c-dfd47e3cca97.png)
![hudtech](https://user-images.githubusercontent.com/37936685/118660395-10f84800-b7e6-11eb-843f-b818e575a971.png)


The tourist icon is really good, lot more creative than the camera which was pretty ambiguous itself.

If you apply my edits i'll merge asap

nmajask committed:
applied grimm-hollowknight requested changes

Adds new signal technician and paramedic hud icons and reverts clerk's back, all requested by grimm-hollowknight

nesisknesis reviewed (approved):
Nothing is bad, these changes are very good.

Whoever made the assistant icon was probably equally perplexed as to what icon it should have had. At least with clerk we have a pre-established symbol.
